pkg/cluster/eks/defaults.go

```diff
@@ -7,8 +7,8 @@ const (
 
 // DefaultImages in each supported location in EC2 (from https://docs.aws.amazon.com/eks/latest/userguide/launch-workers.html)
 var DefaultImages = map[string]string{
-	UsEast1: "ami-0fef2bff3c2e2da93",
-	UsWest2: "ami-0ea01e1d1dea65b5c",
+	UsEast1: "ami-0b2ae3c6bda8b5c06",
```

Comment:
Please add `ami` for GPU (p2, p3) instances.

Reply:
I think we should handle GPU related images in a separate PR.